src/library_dylink.js

```diff
@@ -218,7 +218,11 @@ var LibraryDylink = {
     return function() {
       var sp = stackSave();
       try {
+#if WASM_BIGINT
+        return wbindArray(arguments[0])(Array.prototype.slice.call(arguments, 1));
+#else
         return dynCall(sig, arguments[0], Array.prototype.slice.call(arguments, 1));
+#endif
```

Comment:
I guess this is an example where building with WASM_BIGINT can generate functions with different signatures but in this case that is what we want since we always want the BigInt ABI to be used for `invoke_xxx` calls (since it a completely internal detail its OK that the signatures differ when switching to `WASM_BIGINT`)?